vault/resource_azure_auth_backend_config_test.go

```diff
@@ -97,6 +102,67 @@ func TestAccAzureAuthBackend_wif(t *testing.T) {
 	})
 }
 
+func TestAccAzureAuthBackendConfig_retryFields(t *testing.T) {
```

Comment:
Ideally these test steps could be included in TestAccAzureAuthBackendConfig_basic above. I don't think they warrant creating an entirely new test case.